tests/unit/transformers/test_boolean.py

```diff
@@ -15,8 +15,8 @@ def test___init__(self):
         transformer = BooleanTransformer()
 
         # Asserts
-        self.assertEqual(transformer.nan, -1, "Unexpected nan")
-        self.assertIsNone(transformer.null_column, "null_column is None by default")
+        assert transformer.nan == -1
```

Comment:
@pvk-developer Is deleting all the comments really necessary? Some of them provide useful error messages.

Reply:
My 2 cents: I agree on that having assert comments can be helpful, but I have some doubts about these ones in particular being relevant enough to keep them there, since they basically just repeat what is being asserted. Consider my vote blank here. Your call @pvk-developer @fealho

Reply:
@fealho and @csala I think that the comments in the code while asserting shouldn't be there at all, if we have to write some comments regarding what we are asserting it should be on the docstring of the test itself. My vote goes on this option (move the comments to the docstrings).

Reply:
I agree it didn't matter too much either way, so since the comments have already been deleted I think it can be merged as is.